src/MicrosoftTeams.ts

```diff
@@ -49,23 +49,77 @@ namespace microsoftTeams {
         args?: any[]; // tslint:disable-line:no-any:The args here are a passthrough from OnMessage where we do receive any[]
     }
 
+    /**
+     * Represents information about tabs for an app
+     */
     export interface TabInformation {
         teamTabs: TabInstance[];
     }
 
+    /**
+     * Represents information about a tab instance
+     */
     export interface TabInstance {
+        /**
+         * The name of the tab
+         */
         tabName: string;
+
+        // internal: do not use
```

Comment:
Any particular reason why you aren't using standard jsdoc format for this comment as well?

Reply:
because it was "private" but I can update.